routers/retraining_pipeline.py

```
@@ -0,0 +1,212 @@
+            "total": len(runs),
+            "summary": {
+                "promoted": promoted,
+                "rejected": rejected,
+                "failed": failed,
+            },
+            "runs": recent,
+        }
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)